src/main/scala/nodes/stats/Sampling.scala

```diff
@@ -2,7 +2,7 @@ package nodes.stats
 
 import breeze.linalg.{DenseVector, DenseMatrix}
 import org.apache.spark.rdd.RDD
-import pipelines.{FunctionNode, Transformer}
+import pipelines.FunctionNode
```

Comment:
I was just putting it off to a different PR when we clean up the existing pipelines. If you want me to do it now I can.